doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -378,19 +414,19 @@ MIN_CATALOG_THRESHOLD: int (*default=3*)
     Not currently in use.
 
 MIN_OBSERVABLE_THRESHOLD: int (*default=10*)
-    Not currently in use. The minimum number of observed sources required to continue fitting. If below this threshold, the code will return a status=1 and try with another catalog. 
+    Not currently in use. The minimum number of observed sources required to continue fitting. If below this threshold, the code will return a status=1 and try with another catalog.
 
 MAX_FIT_LIMIT: int (*default=150*)
     Not currently in use.
 
 mosaic_catalog_list: list of strings (*default=["GAIAeDR3", "GSC242", "2MASS"]*)
-    List of available catalogs for aligning for both pipeline and SVM products. The code will go through each catalog in this order. 
+    List of available catalogs for aligning for both pipeline and SVM products. The code will go through each catalog in this order.
 
 mosaic_fit_list: list of strings (*default=["match_relative_fit", "match_2dhist_fit", "match_default_fit"]*)
     List of available fit algorithms for aligning for both pipeline and SVM products; match_default_fit relative alignment without using 2dhist and different throusholds (see json configuration files).
 
 mosaic_fitgeom_list: dict (*default={"rshift": 10, "rscale": 10, "general": 6}*)
-    The different fit geometries tried in alignment as well as their minobj value which specifies the number of matched sources required for a successful fit. For pipeline products, the fitgeometry value is ignored and defaults to a fit geometry of ``rscale``. The fitgeom for the pipeline products is specified as a default in *align_utils.perform_fit*. The value for minobj specified here, however, is used for the pipeline products. 
+    The different fit geometries tried in alignment as well as their minobj value which specifies the number of matched sources required for a successful fit. For pipeline products, the fitgeometry value is ignored and defaults to a fit geometry of ``rscale``. The fitgeom for the pipeline products is specified as a default in *align_utils.perform_fit*. The value for minobj specified here, however, is used for the pipeline products.
```

Comment:
Change "minobj" to minimum number of objects.  Fix this description regarding the type of products and the minimum number of objects.

Reply:
fixed.